### ISSUE TITLE
MAE, MSE loss functions

### ISSUE THREAD

**Issue by muthutt** [tenstorrent-metal/tt-metal#4306]
MAE, MSE loss functions based on PyTorch 
- MAE - https://pytorch.org/docs/stable/generated/torch.nn.L1Loss.html
- MSE - https://pytorch.org/docs/stable/generated/torch.nn.MSELoss.html

**Comment by umadevimcw** [comment]
Merged to main
